Marlin/src/feature/pause.cpp

```diff
@@ -93,16 +93,21 @@ static void ensure_safe_temperature() {
   }
 }
 
+void do_pause_e_move(const float &length, const float fr) {
+  current_position[E_AXIS] += length;
+  set_destination_from_current();
```

Comment:
I suggest 
destination[E_AXIS]=(current_position[E_AXIS] += length); , 
Only one float copying ' for a 8 bits processor it's a big economy of ticks '